org.mixedrealitytoolkit.uxcore/Slider/SliderSounds.cs

```diff
@@ -110,29 +108,16 @@ public class SliderSounds : MonoBehaviour
         public float MinSecondsBetweenTicks { get => minSecondsBetweenTicks; set => minSecondsBetweenTicks = value; }
 
         [SerializeField]
-        [Tooltip("The source of the grab and release sounds")]
-        private AudioSource grabReleaseAudioSource = null;
-
-        /// <summary>
-        /// Gets or sets the AudioSource used to play grab and release sounds.
-        /// </summary>
-        public AudioSource GrabReleaseAudioSource
-        {
-            get => grabReleaseAudioSource;
-            set => grabReleaseAudioSource = value;
-        }
-
-        [SerializeField]
-        [Tooltip("The source of the pass notch sounds")]
-        private AudioSource passNotchAudioSource = null;
+        [Tooltip("The AudioSource to be used when playing slider sounds.")]
+        private AudioSource audioSource = null;
 
         /// <summary>
-        /// Gets or sets the AudioSource used to play sound when the user moves the slider thumb past a notch.
+        /// Gets or sets the AudioSource used to play slider sounds.
         /// </summary>
-        public AudioSource PassNotchAudioSource
+        public AudioSource AudioSource
```

Comment:
nit: do we want to move to using the `[field: SerializeField]` pattern on an auto-property here?

Reply:
Yeah! For example:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/125e8add28d5837150e0a177483a346a3999a486/org.mixedrealitytoolkit.core/Interactables/StatefulInteractable.cs#L95-L96

Reply:
as long as this pattern matches the rest of the file, happy to make the change :)

Reply:
decided to minimize the change to this file and maintain consistency with the existing code pattern